src/main/java/com/khomishchak/ws/controllers/AbstractRestController.java

```diff
@@ -29,4 +30,13 @@ public ResponseEntity<?> handleBalanceNotFoundException(BalanceNotFoundException
         result.put(ex.getClass().getName(), ex.getMessage());
         return ResponseEntity.badRequest().body(result);
     }
+
+    @ExceptionHandler(WhiteBitClientException.class)
+    public ResponseEntity<?> handleBalanceNotFoundException(WhiteBitClientException ex) {
+        Map<String, Object> result = new HashMap<>();
```

Comment:
You can replace it by some `ErrorResponse` class etc. (of course created by you). As I can see you used a map before with key "errors" so would be great to do it.

Reply:
they have different format